docs/RelationalPlaceholders.md

```diff
@@ -0,0 +1,28 @@
+# Velocitab MiniPlaceholders Expansion Documentation
+
```

Comment:
headers aren't needed at the top of docs pages as they're auto-appended based on the file name